pkg/operator/staticpod/startupmonitor/cmd.go

```diff
@@ -165,6 +173,19 @@ func (o *Options) Validate() error {
 	return nil
 }
 
+func setupLogger(logFilePath string) {
+	if len(logFilePath) == 0 {
+		return
+	}
+	klog.SetOutput(&lumberjack.Logger{
+		Filename:   logFilePath,
+		MaxSize:    100, // large files break some editors
+		MaxBackups: 3,
+		MaxAge:     28,   // if something goes wrong we will be called out early, retain logs for ~ a month, just in case
+		Compress:   true, // determines if the rotated log files should be compressed (i.e. gzip)
```

Comment:
do these setting match those of kube-apiserver?

Reply:
except the `Compress` field. `watch-termination` binary [uses](https://github.com/openshift/kubernetes/blob/master/cmd/watch-termination/main.go#L255):
```
l := &lumberjack.Logger{
			Filename:   w.fn,
			MaxSize:    100,
			MaxBackups: 3,
			MaxAge:     28,
			Compress:   false,
		}
```

Reply:
it means that the rotated logs files will be compressed, which should be fine, right ?

Reply:
would not compress. Not super important. But 3x100 is not a lot. Let's just leave it uncompressed.

Reply:
updated to reflect what we use in `watch-termination`